src/main/java/reposense/ConfigParser/Argument.java

```diff
@@ -0,0 +1,52 @@
+package reposense.ConfigParser;
+
+import java.io.File;
+import java.util.Date;
+import java.util.Optional;
+
+public final class Argument {
```

Comment:
if the arguments are literate-able, I think it should be `Configurations` rather than `Argument`